pkg/shp/cmd/buildrun/list.go

```diff
@@ -85,7 +85,10 @@ func (c *ListCommand) Run(params *params.Params, io *genericclioptions.IOStreams
 				break
 			}
 		}
-		age := duration.ShortHumanDuration(time.Since((br.Status.StartTime).Time))
+		age := "0s"
+		if br.Status.StartTime != nil {
+			age = duration.ShortHumanDuration(time.Since((br.Status.StartTime).Time))
+		}
```

Comment:
Given that we call it "age", should we maybe move to `metadata.creationTimestamp` which is always set?

Reply:
I think it makes sense, @SaschaSchwarze0. Given the idea is to report the `BuildRun` age, we can adopt the resource metadata.

However, the "start time" information is also relevant, how about we employ the metadata field to fix the issue at hand (#114), and create another issue for adding the "start time" as a new column?

/cc @HeavyWombat

Reply:
Sounds good. I think we can broaden the scope of the issue. We should generally check which columns we show in `list` calls and which data we show in `get` calls. I don't remember whether we thought about this in great detail initially. But since then we have some more data (like source details and output image digest) and we should revisit what we show.